src/key-value.spec.js

```diff
@@ -0,0 +1,23 @@
+describe('ReactKeyValue', () => {
+  it('should have the proper class name');
+  it('should set the default props');
+  it('should get the initial state from the props');
+  it('should handle adding new rows');
+  it('should handle when a key changes');
+  it('should handle when a value changes');
+  it('should handle removing a row');
+  it('should render a KeyItem');
+  it('should render a ValueItem');
```

Comment:
I would test whether or not it renders the proper amount of elements based on `props.rows`.
And I would also test how it behaves when `props.rows` is an empty array or `undefined`.